### ISSUE TITLE
TypeScript intellisense for i18next not working anymore sinc VSCode update

### ISSUE THREAD

**Issue by adrai** [microsoft/TypeScript#54708]
Type: <b>Bug</b>

1. Clone https://github.com/i18next/i18next
2. Navigate to: https://github.com/i18next/i18next/tree/master/examples/typescript
3. npm i
4. open index.ts
5. check intelisense for => `i18next.t('')`

On VSCode v1.78.2 it works.
On VSCode v1.79.1/v1.79.2 it does not work anymore.

See also: https://github.com/i18next/i18next/issues/1956#issuecomment-1595258734

VS Code version: Code 1.79.2 (695af097c7bd098fbf017ce3ac85e09bbc5dda06, 2023-06-14T08:58:33.551Z)
OS version: Darwin arm64 22.5.0
Modes:

**Comment by pedrodurek** [comment]
Hey @RyanCavanaugh. It seems to be an issue with typescript 5.1. https://github.com/microsoft/TypeScript/issues/54710

**Comment by Risovas** [comment]
Just to let you guys know, I've updated Typescript to `5.1.5`, updated my VSCode again to `1.79.2`, the issue persists. Will go back to `1.78.0` to get the autocompletion working again

**Comment by Bramgus12** [comment]
I think this is also a case of vscode taking over the new typescript version. Did you try vscode insiders?

**Comment by Risovas** [comment]
@Bramgus12 you're right, I don't know why I thought that updating just my project's `typescript` version would solve the issue. With that said, I downloaded the Insiders version (1.80.0) and the issue persists. Guess I'll just keeping trying on a weekly basis until it's fixed

**Comment by Bramgus12** [comment]
Okay, weirdly enough I updated my vscode version to the insiders one and it does work for me now.

**Comment by Risovas** [comment]
I see, well, I mean, it's not an issue that really holding me back or anything, I just notice it when creating a new screen and having to fill up multiple translation keys, or stuff like that. I'll wait for the next Stable version of VSCode and if it still doesn't work, I'll just accept it

**Comment by Bramgus12** [comment]
I just checked again it seemed to be cached lol. So no, it no work in my vscode as well. But I think we just need to wait until vscode updates their ts version. might be a month.

**Comment by Bramgus12** [comment]
seems that it is not in a milestone yet, so I don't think it is in 5.1.5. So we gotta wait until probably the next version
